src/components/blogContent/BlogContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { makeStyles, TextField, Typography } from "@material-ui/core";
+import React from "react";
+const useStyles = makeStyles((theme) => ({
+  "padding-md": {
+    padding: theme.spacing(10),
+  },
+  "padding-sm": {
+    padding: theme.spacing(3),
+  },
+}));
+const BlogContet: React.FC = () => {
+  const classes = useStyles();
+  const contents = ["Title", "Content", "Blurb/Summary", "Tags"];
+  return (
+    <div className={classes["padding-md"]}>
```

Comment:
Referring to these classes as maps via JSON is not as safe as directly accessing the field where your IDE can tell you if the key does/does not exist. Try `classes.padding-md` instead here and see if that works for you. Same on line 17.